cmd/gossamer/configcmd.go

```diff
@@ -44,6 +44,8 @@ import (
 	"github.com/urfave/cli"
 )
 
+var channelSize = 128
```

Comment:
this magic number here is a const ?

Reply:
yes, I was noticing that the node would stall on writing to a channel because it was full, so I added a buffer. it's arbitrary right now